src/scripts/main.js

```diff
@@ -354,7 +354,36 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+function createRow(person) {
+  const row = document.createElement('tr');
+
+  const nameCell = document.createElement('td');
+  const genderCell = document.createElement('td');
+  const bornCell = document.createElement('td');
+  const diedCell = document.createElement('td');
+  const ageCell = document.createElement('td');
+  const centuryCell = document.createElement('td');
+
+  nameCell.textContent = person.name;
```

Comment:
It seems like you have a typo here. You're trying to access 'person.gender' but your object doesn't have a 'gender' property. You have a 'sex' property in your object, perhaps that's what you meant to use?